ArcadiaTeamsBot.ServiceDesk.Requests/RequestTypeForUI.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace ArcadiaTeamsBot.ServiceDesk.Requests
+{
+    using System;
+    using System.Collections.Generic;
+
+    using ArcadiaTeamsBot.ServiceDesk.Abstractions.DTOs;
+
+    public class RequestTypeForUI
+    {
+        private readonly Dictionary<string, string> serviceDeskMappingConfiguration;
+
+        public RequestTypeForUI(Dictionary<string, string> serviceDeskMappingConfiguration)
+        {
+            this.serviceDeskMappingConfiguration = serviceDeskMappingConfiguration;
+            this.RequestTypeForUIFields = new List<RequestTypeForUIField>();
+        }
+
+        public List<RequestTypeForUIField> RequestTypeForUIFields { get; set; }
+
+        public void LoadFieldsTypes(ServiceDeskRequestTypeDTO requestTypeDTO)
```

Comment:
Instead all of this it would better be a factory which will manage dependencies and create types for UI by specified request type DTO.

Reply:
Implemented factory method